apps/emqx_bridge/src/emqx_bridge.erl

```diff
@@ -339,17 +339,21 @@ flatten_confs(Conf0) ->
 do_flatten_confs(Type, Conf0) ->
     [{{Type, Name}, Conf} || {Name, Conf} <- maps:to_list(Conf0)].
 
-get_matched_bridges(Topic) ->
+get_matched_egress_bridges(Topic) ->
     Bridges = emqx:get_config([bridges], #{}),
     maps:fold(
-        fun(BType, Conf, Acc0) ->
-            maps:fold(
-                fun(BName, BConf, Acc1) ->
-                    get_matched_bridge_id(BType, BConf, Topic, BName, Acc1)
-                end,
-                Acc0,
-                Conf
-            )
+        fun
+            (mqtt, #{ingress := _}, Acc0) ->
```

Comment:
Actually, I just remembered... A MQTT bridge can be _both_ egress and ingress, right?

Maybe it's better to assert that `egress` is _absent_ from the config to skip?

Reply:
true it is possible to create a bridge with both ingress and egress.